Servers/UI/OJS.Servers.Ui/Controllers/Api/SubmissionsController.cs

```diff
@@ -72,6 +72,26 @@ public async Task<IActionResult> GetForProfile()
             .MapCollection<SubmissionResultsResponseModel>()
             .ToOkResult();
 
+    /// <summary>
+    /// Gets a subset of submission results for the selected user by specific problem and given take count.
+    /// </summary>
+    /// <param name="submissionId">The id of the submission.</param>
+    /// <param name="problemId">The id of the problem.</param>
+    /// <param name="isOfficial">Should the submissions be only from compete mode.</param>
+    /// <param name="take">Number of submissions to return.</param>
+    /// <returns>A collection of submissions for a specific problem.</returns>
+    [HttpGet("{submissionId:int}/{problemId:int}")]
+    [ProducesResponseType(typeof(IEnumerable<SubmissionResultsResponseModel>), Status200OK)]
+    public async Task<IActionResult> GetSubmissionDetailsResults(
+        int submissionId,
+        int problemId,
```

Comment:
Why is the problemId needed if we have the submissionId? 

Also the request url in which this results is` /Submissions/GetSubmissionDetailsResults/23668547/9145?isOfficial=false&take=4`

https://retool.com/blog/best-practices-for-building-crud-endpoints/. In this example `/GetSubmissionDetailsResults/23668547/9145` its not very clear what the 2nd id value is for. If you start stacking ids you probably need a model for this.